src/windows/room/chat.rs

```diff
@@ -202,29 +203,42 @@ impl ChatState {
                         },
                     };
 
-                    if !force && filename.exists() {
-                        let msg = format!(
-                            "The file {} already exists; use :download! to overwrite it.",
-                            filename.display()
-                        );
-                        let err = UIError::Failure(msg);
+                    if filename.exists() {
+                        if !flags.intersects(DownloadFlags::FORCE | DownloadFlags::OPEN) {
```

Comment:
Check just for `DownloadFlags::FORCE` here, so that `:open` doesn't accidentally overwrite an already existing file.